### DISCUSSION TITLE
Markdown/discount explanation

### DISCUSSION THREAD

**Discussion by vbugaichuk** [sibylhe/mmm_stan#46] [Q&A] [answered]
Hello,

I have some questions about markdown varibles in your data. What do they represent and how they were calculated? You have mrkdn_valadd_edw and mrkdn_pdm. For example, I have many discounts in one time with different % and different products they can be used on. How can I aggregate all this discounts to one or some variables on weekly and national level of data? 

Thanks for attention!

**Comment by sibylhe** [comment, marked as answer]
In this dataset, mrkdn_valadd_edw = markdown of value-added service, mrkdn_pdm = markdown of products. It's a weighted average of all discounts. For example, last week sales:
product    original_price    actual_price    units
A               $10                    $8                     100
B               $20                    $15                   200
markdown = (8 * 100 + 15 * 200)/(10 * 100 + 20 * 200)
